services/sasquatch/charts/strimzi-kafka/values.yaml

```diff
@@ -0,0 +1,45 @@
+# Default values for kafka used in Sasquatch.
+cluster:
+  # -- Name used for the Kafka cluster, and used by Strimzi for many annotations.
+  name: sasquatch
+
+kafka:
+  # -- Version of Kafka to deploy.
+  version: "3.0.0"
+  # -- Number of Kafka broker replicas to run.
+  replicas: 3
+  storage:
+    # -- Size of the backing storage disk for each of the Kafka brokers.
+    size: 100Gi
+    # -- Name of a StorageClass to use when requesting persistent volumes.
+    storageClassName: ""
+  # -- Configuration overrides for the Kafka server.
+  config:
+    # -- Number of minutes for a consumer group's offsets to be retained.
+    offsets.retention.minutes: 10080
+    # -- Number of days for a topic's data to be retained.
+    log.retention.hours: 168
+    # -- Maximum retained number of bytes for a topic's data.
+    log.retention.bytes: "644245094400"
```

Comment:
Do you want this much nested hierarchy? In other words, why not `offsetsRetentionMinutes`, etc.?

Reply:
Yeah, those are Kafka configuration options, I cannot change that. What is under the config section overrides the default kafka configuration which is written in a `ConfigMap`.